### PR TITLE
DeflateStream decompression app compat issue

### PR DESCRIPTION
**Pull request by rpetrusha**
/cc @PriyaPurkayastha, @krwq

### CODE REVIEW

File: Documentation/compatibility/deflatestream-uses-native-apis-for-decompression.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+
+Starting with apps that target the .NET Framework 4.7.1, the implementation of decompression in the `T:System.IO.Compression.DeflateStream` class has changed to use native Windows APIs by default. This results in an average performance improvement of approximately 300%.
```
Comment (krwq) [inline Documentation/compatibility/deflatestream-uses-native-apis-for-decompression.md]: .NET Framework 4.7.2

File: Documentation/compatibility/deflatestream-uses-native-apis-for-decompression.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <runtime>
+       <AppContextSwitchOverrides
+          value="Switch.System.IO.Compression.DoNotUseNativeZipLibraryForDecompression=true" />
+    </runtime>
+</configuration>
+```
+### Affected APIs
+`T:System.IO.Compression.DeflateStream`
```
Comment (krwq) [inline Documentation/compatibility/deflatestream-uses-native-apis-for-decompression.md]: also `T:System.IO.Compression.GZipStream` which implementation is in DeflateStream

File: Documentation/compatibility/deflatestream-uses-native-apis-for-decompression.md (hunk 1 above)
Comment (krwq) [inline Documentation/compatibility/deflatestream-uses-native-apis-for-decompression.md]: Also improvement is only for decompression (DeflateStream despite confusing name of this class)

**Comment by rpetrusha** [comment]
Thanks, @krwq. I've addressed your comments.

**Comment by rpetrusha** [comment]
@PriyaPurkayastha and @krwq, I've updated the topic based on Priya's comments.

**Pull request merged**